### PR TITLE
V22 search suggestion

### PR DESCRIPTION
**Pull request by levogirar**
Added homepage suggestion
Hid the company edit
Minor visual changes

### CODE REVIEW

File: WebRoot/homepage/view result.php [hunk 1]
```diff
@@ -52,26 +52,30 @@ class="button-filter"
 				<h2><?php echo TEXT[12] ?></h2>
 			</div>
 			<div class="search-result-box">
-				<p>Showing <?php echo $companies['Counter'] ?> results for</p>
+				<p><?php echo TEXT[15] . $companies['Counter'] . TEXT[16] ?></p>
```
Comment (Nils85) [inline WebRoot/homepage/view result.php:55]: Use `printf()` with only **one** translation line instead, like with TEXT[1] in `WebRoot/company/text en.php` and `WebRoot/company/view.php`

File: WebRoot/company/view.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 	<div class="container">
 		<section class="column-left">
 			<div class="company-sticky">
-				<div class="box-panel" id="company-panel">
+				<div class="box-panel" id="company-panel" style="display: flex;">
```
Comment (Nils85) [inline WebRoot/company/view.php:31]: Semicolon is useless for only one property

File: WebRoot/company/view.php [hunk 3]
```diff
@@ -100,25 +100,24 @@
 							<?php endif ?>
 						</ul>
 					</section>
-					<hr>
 					<?php if (isset($user) && $user->isAdmin()): ?>
-					<!-- Not sure if this make sense -->
-					<a href="<?php echo $_SERVER["HTTP_HOST"] . $_SERVER["REQUEST_URI"];?>/?edit">
+					<hr>
+					<a onclick="companyEdit();">
```
Comment (Nils85) [inline WebRoot/company/view.php:105]: Same here, semicolon is useless for only one JavaScript function call

File: WebRoot/static/dhtml/company.js [hunk 4]
```diff
@@ -3,22 +3,32 @@
  * https://usefulangle.com/post/118/javascript-intersection-observer
  */
 
-var panel = document.getElementById("company-panel");
-var summary = document.getElementById("mobile-summary");
+const panel = document.getElementById("company-panel");
+const edit = document.getElementById("edit");
+const summary = document.getElementById("mobile-summary");
```
Comment (Nils85) [inline WebRoot/static/dhtml/company.js:8]: `const` comes from ES6 so always use `var` to stay compatible with ES5 (see CONTRIBUTING.md -> JavaScript)

**Review by Nils85 — CHANGES_REQUESTED** [review]

File: WebRoot/static/dhtml/company.js (hunk 4 above)
Comment (levogirar) [reply WebRoot/static/dhtml/company.js]: Maybe we should use ES6 🙊 it came out in 2015! It is just that most tutorials I find uses ES6

**Review by Nils85 — APPROVED** [review]

**Pull request merged**